mediapipe/tasks/python/vision/hand_landmarker.py

```diff
@@ -0,0 +1,381 @@
+_IMAGE_OUT_STREAM_NAME = 'image_out'
+_IMAGE_TAG = 'IMAGE'
+_NORM_RECT_STREAM_NAME = 'norm_rect_in'
+_NORM_RECT_TAG = 'NORM_RECT'
+_HANDEDNESS_STREAM_NAME = 'handedness'
+_HANDEDNESS_TAG = 'HANDEDNESS'
+_HAND_LANDMARKS_STREAM_NAME = 'landmarks'
+_HAND_LANDMARKS_TAG = 'LANDMARKS'
+_HAND_WORLD_LANDMARKS_STREAM_NAME = 'world_landmarks'
+_HAND_WORLD_LANDMARKS_TAG = 'WORLD_LANDMARKS'
+_TASK_GRAPH_NAME = 'mediapipe.tasks.vision.hand_landmarker.HandLandmarkerGraph'
+_MICRO_SECONDS_PER_MILLISECOND = 1000
+
+
+@dataclasses.dataclass
+class HandLandmarksDetectionResult:
+  """The hand landmarks detection result from HandLandmarker, where each vector
+  element represents a single hand detected in the image.
+
+  Attributes:
+    handedness: Classification of handedness.
+    hand_landmarks: Detected hand landmarks in normalized image coordinates.
+    hand_world_landmarks: Detected hand landmarks in world coordinates.
+  """
+
+  handedness: List[List[category_module.Category]]
+  hand_landmarks: List[List[landmark_module.NormalizedLandmark]]
+  hand_world_landmarks: List[List[landmark_module.Landmark]]
+
+
+def _build_detection_result(
+    output_packets: Mapping[str,packet_module.Packet]
+) -> HandLandmarksDetectionResult:
+  """Constructs a `HandLandmarksDetectionResult` from output packets."""
+  handedness_proto_list = packet_getter.get_proto_list(
+      output_packets[_HANDEDNESS_STREAM_NAME])
+  hand_landmarks_proto_list = packet_getter.get_proto_list(
+      output_packets[_HAND_LANDMARKS_STREAM_NAME])
+  hand_world_landmarks_proto_list = packet_getter.get_proto_list(
+      output_packets[_HAND_WORLD_LANDMARKS_STREAM_NAME])
+
+  handedness_results = []
+  for proto in handedness_proto_list:
+    handedness_categories = []
+    handedness_classifications = classification_pb2.ClassificationList()
+    handedness_classifications.MergeFrom(proto)
+    for handedness in handedness_classifications.classification:
+      handedness_categories.append(
+          category_module.Category(
+              index=handedness.index,
+              score=handedness.score,
+              display_name=handedness.display_name,
+              category_name=handedness.label))
+    handedness_results.append(handedness_categories)
+
+  hand_landmarks_results = []
+  for proto in hand_landmarks_proto_list:
+    hand_landmarks = landmark_pb2.NormalizedLandmarkList()
+    hand_landmarks.MergeFrom(proto)
+    hand_landmarks_results.append([
+        landmark_module.NormalizedLandmark.create_from_pb2(hand_landmark)
+        for hand_landmark in hand_landmarks.landmark
+    ])
+
+  hand_world_landmarks_results = []
+  for proto in hand_world_landmarks_proto_list:
+    hand_world_landmarks = landmark_pb2.LandmarkList()
+    hand_world_landmarks.MergeFrom(proto)
+    hand_world_landmarks_results.append([
+        landmark_module.Landmark.create_from_pb2(hand_world_landmark)
+        for hand_world_landmark in hand_world_landmarks.landmark
+    ])
```

Comment:
ditto. This will trigger the "Comprehension too complex" warning. Please use for loop instead.